README.txt

```diff
@@ -58,30 +87,28 @@ some data for testing and some simulator outputs]
 	INCLUDECADMIUM=-I ../../cadmium/include
     Update the relative path to cadmium/include from the folder where the makefile is. You need to take into account where you copied the folder during the installation process
 	Example: INCLUDECADMIUM=-I ../../cadmium/include
-    
-2 - Run the unit tests
-	2.1. Run subnet test
-		1 - Open the terminal. Press in your keyboard Ctrl+Alt+t
-		2 - Set the command prompt in the test/subnet folder. To do so, type in the terminal the path to this folder.
-			Example: cd Documents/AlternateBitProtocol/test/subnet
-		3 - To compile the test, type in the terminal:
-			make clean; make all
-		4 - To run the test, type in the terminal "./NAME_OF_THE_COMPILED_FILE". For this specific test you need to type:
-			./SUBNET

```

Comment:
Should be ./SUBNET_TEST

Reply:
This is in a red color and it refers to the old readme file.